docs/docusaurus/versioned_docs/version-1.5.0/lte/upgrade_1_5.md

```diff
@@ -0,0 +1,29 @@
+---
+id: version-1.5.0-upgrade_1_5
+title: Upgrade to v1.5
+hide_title: true
+original_id: upgrade_1_5
+---
+
+# Upgrade to v1.5
+
+---
+> **_NOTE:_** Please note that Fuji is the last release with support for Debian.
+---
+
+### Repo Change
+
+In Magma Fuji (v1.5), Magma artifacts are now hosted on the new Magmacore repositories.
+(https://artifactory.magmacore.org/)
```

Comment:
`...are now hosted on the new Magmacore repositories at [artifactory.magmacore.org](https://artifactory.magmacore.org/).`